components/sections/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,95 @@
+// React/Relay
+import React, { PureComponent } from "react";
+import { createFragmentContainer, graphql } from "react-relay";
+import singleFragmentComponent from "../../../lib/singleFragmentComponent";
+import classnames from "classnames";
+
+// Component
+import { Props } from "./Tabs.types";
+import styles from "./Tabs.scss";
+
+// External
+import Text from "../../text";
+
+export class Tabs extends PureComponent<Props, { selectedTab: number }> {
+  constructor(props) {
+    super(props);
+    this.state = { selectedTab: 0 };
+  }
+
+  private selectTab(index: number): void {
+    this.setState({ selectedTab: index });
+  }
+
+  private tabNameColor(index: number) {
+    const { tabs_name_color, tabs_name_selected_color } = this.props;
+    const { selectedTab } = this.state;
+
+    if (selectedTab === index) {
+      return tabs_name_selected_color ?? "#df8832";
+    }
+
+    return tabs_name_color ?? "#81a88f";
+  }
+
+  public render() {
+    const { background_color, tabs_content_color, tabs_list } = this.props;
+    const { selectedTab } = this.state;
+
+    return (
+      <section
+        className={styles.tabs}
+        style={{ backgroundColor: background_color ?? "#31683D" }}
+      >
+        <div className={styles.tabsName}>
+          {tabs_list.map((tab, index) =>
+            <div
+              className={classnames(styles.tabName, {
+                [styles.selectedTabName]: selectedTab === index
+              })}
+              style={{
+                color: this.tabNameColor(index),
+                borderBottomColor:
+                  selectedTab === index ? this.tabNameColor(index) : undefined,
+              }}
+              onClick={() => this.selectTab(index)}
+            >
+              {tab.name}
+            </div>
+          )}
+        </div>
+        <div className={styles.tabsContent}>
+          {tabs_list.map((tab, index) =>
+            <div
+              className={styles.tabTitle}
+              style={{
+                color: tabs_content_color ?? "#ffffff",
```

Comment:
Should we use a class "active" for this?

Reply:
I've decided to leave the tabs content, name and background color customizable so they can change it at will. The customization prevents us from having predefined classes :(

Reply:
I don't understand how putting this in a class prevents us to change it at will. The reason is to keep a single source to look styles in.

Reply:
I've made the section totally customizable in a color level. The prismic user can select the different colors at will.
Are you saying that we should put the default into classes?